pkg/gateway/operations/listobjects.go

```diff
@@ -393,3 +400,57 @@ func (controller *ListObjects) Handle(w http.ResponseWriter, req *http.Request,
 		_ = o.EncodeError(w, req, nil, gatewayerrors.Codes.ToAPIErr(gatewayerrors.ErrBadRequest))
 	}
 }
+
+func handleListMultipartUploads(w http.ResponseWriter, req *http.Request, o *RepoOperation) {
+	query := req.URL.Query()
+	maxUploadsStr := query.Get(QueryParamMaxUploads)
+	opts := block.ListMultipartUploadsOpts{}
+	if maxUploadsStr != "" {
+		maxUploads, err := strconv.ParseInt(maxUploadsStr, 10, 32)
+		if err != nil {
+			o.Log(req).WithField("MaxParts", maxUploadsStr).
+				WithError(err).Error("malformed query parameter 'MaxParts'")
+			_ = o.EncodeError(w, req, err, gatewayerrors.Codes.ToAPIErr(gatewayerrors.ErrInternalError))
```

Comment:
- Verify that this is the response from aws's s3 service? usually it is BadRequest.
- Same for using invalid value (not 1-1000) for this parameter